doc/source/index.rst

```diff
@@ -95,7 +95,7 @@ Documentation Index
    Custom Metrics <analytics/custom_metrics.md>
    Payload Logging with ELK <analytics/logging.md>
    Distributed Tracing with Jaeger <graph/distributed-tracing.md>
-   Autoscaling in Kubernetes <graph/autoscaling.md>
```

Comment:
This is not something critical, just a thought. 

At some point in the future, would it make sense to consider redirects whenever we remove / rename paths? For example, in this case:

- https://docs.seldon.io/projects/seldon-core/en/latest/graph/autoscaling.html => https://docs.seldon.io/projects/seldon-core/en/latest/graph/scaling.html

Is this something supported by `readthedocs`?

Reply:
we can add to https://readthedocs.org/dashboard/seldon-core/redirects/ when published

Reply:
That's great to know! I didn't know we had that available.